health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/WorkflowCommentPopUp.js

```
@@ -106,59 +102,64 @@ const WorkflowCommentPopUp = ({ onClose, heading, submitLabel, url, requestPaylo
-                    {error && (
-                        <ErrorMessage
-                            message={t('HCM_MICROPLAN_ADD_COMMENT_REQUIRED')}
-                            truncateMessage={true}
-                            maxLength={256}
-                            showIcon={true}
+        <>
+            <PopUp
+                style={{ width: "700px" }}
+                onClose={onClose}
+                heading={t(heading)}
+                children={[
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)